README.md

```diff
@@ -70,8 +70,8 @@ Read on to see what else Wallaby can do or check out the [Official Documentation
 Add Wallaby to your list of dependencies in `mix.exs`:
 
 ```elixir
-def deps do
-  [{:wallaby, "~> 0.19.1"}]
+def do
```

Comment:
you lost the `deps` somewhere along the way

Reply:
d'oh! That's what I get for editing from the Github UI. 😆